### ISSUE TITLE
send Slack notification if e2e fails/on proceed or abort prompt

### ISSUE THREAD

**Issue by vdice** [deis/workflow-cli#127]
The e2e [stage](https://github.com/deis/workflow-cli/blob/master/Jenkinsfile#L136) in the workflow-cli pipeline currently awaits (indefinitely) for input if the first e2e run failed, asking the user whether to proceed (with another e2e run) or abort.  It would be helpful if a Slack message was broadcast, perhaps to #testing, ideally pinging the owner of the PR, on this event.

**Comment by Joshua-Anderson** [comment]
This is contingent on updating the slack plugin and the jobs-dsl plugin to get slack message support in jenkinsfiles.

**Comment by Joshua-Anderson** [comment]
#133 Uses email notifications to slack for a temporary stopgap measure instead of actual slack messages.

**Comment by vdice** [comment]
As @Joshua-Anderson mentioned, we've set up email notifications for now.  We can re-open and/or create a new ticket to change over to notifications when necessary.  Closing.
